sql/core/src/test/scala/org/apache/spark/sql/execution/vectorized/ColumnVectorSuite.scala

```diff
@@ -243,6 +243,93 @@ class ColumnVectorSuite extends SparkFunSuite with BeforeAndAfterEach {
     assert(testVector.getArray(3).toIntArray() === Array(3, 4, 5))
   }
 
+  testVectors("array append", 1, arrayType) { testVector =>
```

Comment:
This is a bug fix, so could you add the prefix: `SPARK-35898:` in the test names?

Reply:
Sure, I will add them! I did not add before them since they test general usage patterns instead of a specific edge case.